src/Controls/tests/UITests/Tests/Issues/_IssuesUITest.cs

```diff
@@ -9,15 +10,41 @@ public abstract class _IssuesUITest : UITestBase
 
 		protected override void FixtureSetup()
 		{
-			base.FixtureSetup();
-			NavigateToIssue(Issue);
+			int retries = 0;
+			while (true)
+			{
+				try
+				{
+					base.FixtureSetup();
+					NavigateToIssue(Issue);
+					break;
+				}
+				catch (Exception)
+				{
+					if (retries++ < 1)
+					{
+						TestContext.Error.WriteLine("There was a problem with the FixtureSetup retrying");
+						Reset();
+					}
+					else
+					{
+						throw;
+					}
+				}
+			}
 		}
 
 		protected override void FixtureTeardown()
 		{
 			base.FixtureTeardown();
-			App.NavigateBack();
-			App.Tap("GoBackToGalleriesButton");
+			try
+			{
+				App.NavigateBack();
+				App.Tap("GoBackToGalleriesButton");
+			}
+			catch (Exception)
+			{ 
```

Comment:
Should we log it ?

Reply:
Yes that is a good idea, I will add an error message for this as well.